server.js

```diff
@@ -81,6 +89,7 @@ function join_custom_room(ws, join_room_json) {
     set_name(player, join_room_json)
   }
 
+  // Get the room id from the passed in json.
```

Comment:
I don't particularly like comments like this that just say exactly what the code says.
var room_id = join_room_json.room_id.trim()
seems pretty clear

Reply:
I find it easier when scanning to read brief summaries of the purpose of each block of code, then only read the code itself if I have a reason to - it turns a function into a story you can skim. but I do not mind removing this

Reply:
This particular comment isn't a summary of a block of code though, it's literally rewriting the next line in english.
I think if you want to put in a summary of the block of code, that's fine.
I just don't want like a comment explaining every single line when the line pretty much says that.